### PR TITLE
 #164798223 The user should able to see their reading stats

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Enables user to view their reading stats

#### Description of Task to be completed?
The endpoint will be able to fetch the most recently reads by the user

#### How should this be manually tested?
- Fetch this branch `git fetch origin ft-see-reading-stats-164798223`
- Checkout to the branch `git checkout ft-see-reading-stats-164798223`
- Run `pipenv shell`
- Run `python manage.py migrate`
- Run `python manage.py runserver`

#### What are the relevant pivotal tracker stories?
 [#164798223](https://www.pivotaltracker.com/story/show/164798223)

#### Screenshots (if appropriate)
Screenshot 2019-04-15 at 16 04 09

#### Postman Collection
[Run in Postman](https://app.getpostman.com/run-collection/6d1b648c3e9a9b00e07b)

### CODE REVIEW

File: authors/apps/articles/views.py [hunk 1]
```diff
@@ -61,6 +61,18 @@ class RetrieveUpdateDestroyArticle(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = serializers.ArticleSerializer
     lookup_field = 'slug'
 
+    def get_serializer_context(self):
```
Comment (ja-odur) [inline authors/apps/articles/views.py]: @ezrogha please try removing this method/function to see if the tests can pass without overriding it. I feel the default unridden implementation will work as well.

Comment (ezrogha) [reply authors/apps/articles/views.py]: You are right @ja-odur, tests are actually passing

**Review by ja-odur — CHANGES_REQUESTED** [review]
@ezrogha for read-stats more than 10, the recent ones are not displaced. The endpoint simply returns the first 10 instead of the last 10
Please adjust your response body to look like
```
{
  "read_stats": []
}
```

**Comment by ezrogha** [comment]
> @ezrogha for read-stats more than 10, the recent ones are not displaced. The endpoint simply returns the first 10 instead of the last 10
> Please adjust your response body to look like
> 
> ```
> {
>   "read_stats": []
> }
> ```

Thank you @ja-odur, Your feedback was very Insightful, I have worked on it
Kindly review my PR again,

**Review by ja-odur — APPROVED** [review]
LGTM

**Review by felixkiryowa — APPROVED** [review]

**Review by dannylwe — APPROVED** [review]

**Pull request merged**